packages/bits/demo/src/components/demo/tabgroup/tab-heading-group-with-router/tab-heading-group-with-router.example.component.html

```diff
@@ -1,10 +1,16 @@
 <div>
     <nui-tab-heading-group>
-        <nui-tab-heading *ngFor="let tab of tabsetContent" [tabId]="tab.id" routerLink="/tabgroup/{{tab.id}}" [active]="currentTabRoute === tab.id">
+        <nui-tab-heading
```

Comment:
style-wise the convention we use is to have the first attribute on the same line as the opening element tag

Reply:
Hi Blake, I wonder if we need to update our style guide .md to include stylistic things like these. I didn't find any mentions of this in the style guide. It may be useful for the newcomers

Reply:
fixed it

Reply:
Usually we fall back to google style guides for things like this, but I don't see anything explicit [here](https://google.github.io/styleguide/htmlcssguide.html) about it. So, I'll add it to the markdown. Thanks.

Reply:
I've added it [here](https://github.com/solarwinds/nova/blob/bugfix/NUI-5964-If-start-of-daylight-saving-time-is-within-the-x-domain,-TimeIntervalScale-intervals-produce-narrow-bars/docs/STYLE_GUIDE.md#html-formatting). It'll be part of my next PR.